console_windows.go

```diff
@@ -154,6 +154,10 @@ func (m *master) Fd() uintptr {
 	return m.in
 }
 
+func (m *master) Name() string {
+	return ""
+}
```

Comment:
@mlaventure would have a good idea what the windows side needs.  its actually very complex because they use all 3 stdio for the console ( i think )